NDB.Covid19/NDB.Covid19.Droid/Resources/drawable/text_box_background.xml

```diff
@@ -0,0 +1,11 @@
+﻿<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:bottomLeftRadius="10dp" android:bottomRightRadius="10dp" android:topLeftRadius="10dp" android:topRightRadius="10dp" />
+            <stroke
```

Comment:
No border in figma, should it be here?
(very small thing above, figma uses border radius 8)